package.json

```diff
@@ -12,6 +12,9 @@
 
   "dependencies": {
     "babel": "^5.8.23",
+    "bluebird": "^2.10.2",
```

Comment:
In my opinion 0.10 is feeling pretty ancient at the moment, and now that Node has an official stable production version, I'd be happy if we could go 4+, but that might be unreasonable. I'm fine with dropping 0.10, but if that's the case, why favor 0.12 over 4?

I could easily be swayed any direction on this, no major opinion.